### ISSUE TITLE
Can't delete a file. Always return unauthorized

### ISSUE THREAD

**Issue by adihanifsdr** [BunnyWay/BunnyCDN.Java.Storage#13]

**Comment by doghouch** [comment]
Hello,

Thank you for reporting the issue; do the other API methods work with your existing API key?

**Comment by adihanifsdr** [comment]
Yes, the other (BunnyCDN API PHP) is working with my existing API Key. Looks like there any update on BunnyCDN Server End?

**Comment by doghouch** [comment]
Hello,

I'll investigate this further then, would you mind providing the "path" that you were using? Feel free to censor out/omit any private information.
